defaults/main.yml

```
@@ -3,6 +3,9 @@
 # The provider for nbde_client role. Currently 'clevis' is supported.
 nbde_client_provider: clevis
 
+# Deploy mechanisms for opening the volume on boot
+nbde_early_boot: true
```

```suggestion
nbde_client_early_boot: true
```
to match docs in README and variable naming conventions